common/static/common/js/spec/components/paginated_view_spec.js

```diff
@@ -0,0 +1,182 @@
+define([
+    'backbone',
+    'underscore',
+    'common/js/spec_helpers/ajax_helpers',
+    'common/js/components/views/paginated_view',
+    'common/js/components/collections/paging_collection'
+], function (Backbone, _, AjaxHelpers, PaginatedView, PagingCollection) {
+    'use strict';
+    describe('PaginatedView', function () {
+        var TestCardView = Backbone.View.extend({
+            className: 'test-card',
+            tagName: 'div',
+            initialize: function () {
+                this.render();
+            },
+            render: function () {
+                this.$el.text(this.model.get('text'));
+                return this;
+            }
+        }),
+            TestPaginatedView = PaginatedView.extend({type: 'test', itemViewClass: TestCardView}),
+            testCollection,
+            testView,
+            initialItems,
+            nextPageButtonCss = '.next-page-link',
+            previousPageButtonCss = '.previous-page-link',
+            generateItems = function (startIndex, stopIndex) {
+                return _.map(_.range(startIndex, stopIndex + 1), function (i) {
+                    return {text: 'card ' + i};
+                });
+            };
+
+        beforeEach(function () {
+            setFixtures('<div class="test-container"></div>');
+            initialItems = generateItems(1, 5);
```

Comment:
I addressed this by removing the first argument, which was providing no value to the tests. `generateItems` now just takes a single numerical argument: the number of items to create.